.circleci/config.yml

```diff
@@ -2,12 +2,12 @@ version: 2.1
 orbs:
   browser-tools: circleci/browser-tools@1.2.5
   node: circleci/node@5.0.2
-  ruby: circleci/ruby@1.6.0
+  ruby: circleci/ruby@2.0.0
```

Comment:
🗒️ The new Ruby 3.1.3 image is apparently incompatible with the native extensions stored in the bundler cache from previous CI jobs:

> LoadError: libssl.so.1.1: cannot open shared object file: No such file or directory - /home/circleci/project/vendor/bundle/ruby/3.1.0/gems/puma-5.6.5/lib/puma/puma_http11.so

I resolved this by upgrading to the latest version of the Ruby orb (which was something we needed to do anyway). This caused the problematic bundler cache to get invalidated.